app/views/settings/email/_email_fieldset.html.erb

```diff
@@ -1,4 +1,6 @@
 <%= f.govuk_fieldset(legend: { text: f.object.class.human_attribute_name(:email_fieldset) }) do %>
+
+  <% if ENV['REPLY_TO'] == 'disabled' %>
```

Comment:
I'd probably do `if <feature flag>  <new code> else <existing code>` as this covers the case where the feature flag is omitted or set incorrectly

Reply:
all done